SermonTableViewController.m

```diff
@@ -113,32 +114,29 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
         cell = [nib objectAtIndex:0];
     }
     Sermon *sermon = [sermons objectAtIndex:indexPath.row];
-    
     cell.title.text = [sermon getTitle];
     cell.title.adjustsFontSizeToFitWidth = YES;
     [cell.title sizeToFit];
-    
-    cell.subtitle.text = [sermon getVerse];
-    [cell.subtitle sizeToFit];
-    
-    cell.date.text = [sermon getDateInString];
+    cell.verse.text = [sermon getVerse];
+    cell.verse.adjustsFontSizeToFitWidth = YES;
+    [cell.verse sizeToFit];
+    cell.series.text = [sermon getSeries];
```

Comment:
Can we add new line after each cell settings?
So do all `cell.verse` stuff and then new line and then `cell.series` stuff, and so on